### DISCUSSION TITLE
I cannot find the GPT-4o-mini model listed in the model settings after adding my OpenAI API key.

### DISCUSSION THREAD

**Discussion by olbboy** [langgenius/dify#6518] [Help] [answered]
# Discussion: GPT-4o-mini Model Availability in Dify Model Settings

## Issue Description
I am currently integrating the OpenAI API into Dify and attempting to set up the GPT-4o-mini model. However, I cannot find the GPT-4o-mini model listed in the model settings after adding my OpenAI API key.

## Details
- **Platform:** Dify
- **API Provider:** OpenAI
- **Model in Question:** GPT-4o-mini
- **Issue Encountered:** GPT-4o-mini model not appearing in the Dify model settings

## Steps to Reproduce
1. Navigate to the model settings in Dify.
2. Add the OpenAI API key.
3. Look for the GPT-4o-mini model in the available models list.

## Expected Behavior
The GPT-4o-mini model should appear in the list of available models after adding the OpenAI API key.

## Actual Behavior
The GPT-4o-mini model does not appear in the list of available models.

## Questions
1. Is this issue because OpenAI has not yet released the GPT-4o-mini model for public use?
2. Alternatively, is it possible that Dify has not yet updated its system to support the GPT-4o-mini model?

## Additional Context
I have confirmed that my OpenAI API key is valid and working for other models like GPT-3.5 and GPT-4. I have also checked the OpenAI documentation, and there is mention of GPT-4o-mini, but it is unclear whether it is fully available or if there are specific integration steps needed for Dify.

## Screenshots
![Screen Shot 2024-07-22 at 12 27 10](https://github.com/user-attachments/assets/f438a71f-2464-41d2-bffa-b2abbc2ecada)

## References
- [OpenAI API Documentation](https://platform.openai.com/docs)
- [Dify Documentation](https://docs.dify.ai)

Any insights or assistance on this matter would be greatly appreciated!

**Comment by zhangzw16** [comment, marked as answer]
check [here](https://github.com/langgenius/dify/issues/6507#issuecomment-2241816448). Seems it has not yet been supported if you are using docker version.

**Reply by crazywoola** [reply]
Please wait for the new version, it will be introduced in v0.6.15

**Reply by olbboy** [reply]
Thank you @crazywoola, hope to use the v0.6.15 version soon

**Reply by crazywoola** [reply]
Yes we will release it in this week
